src/__tests__/fs.spec.js

```diff
@@ -46,3 +50,17 @@ it('makeDirs() should create multiple folders', () => {
 	expect(mkdirp.sync).toBeCalledWith('a');
 	expect(mkdirp.sync).toBeCalledWith('b');
 });
+
+it('deleteFiles() should delete multiple folders', () => {
```

Comment:
folders → files.

Reply:
do you think, in the future, would be interesting to use fixtures like the tests from https://github.com/sindresorhus/del/blob/master/test.js?

Reply:
Not sure it would help here (that would require a lot of refactoring anyway) but we want something like that to tests tasks in Mrm — https://github.com/sapegin/mrm/issues/15